config/jobs/kubernetes-sigs/sig-windows/sig-windows-config.yaml

```diff
@@ -0,0 +1,76 @@
+presets:
+- labels:
+    preset-azure-acsengine: "true"
+  env:
+  - name: AZURE_CREDENTIALS
+    value: /etc/azure-cred/credentials
+  - name: AZURE_SSH_PUBLIC_KEY_FILE
+    value: /etc/azure-ssh/azure-ssh-pub
+  - name: CLOUD_CONFIG
+    value: random 
+  - name: AZ_STORAGE_CONTAINER_NAME
+    value: mystoragecontainer
+  - name: REGISTRY
+    value: gcr.io/win-e2e-test/win-e2e-hyperkube
+  - name: WIN_BUILD
+    value: https://raw.githubusercontent.com/Azure/acs-engine/master/scripts/build-windows-k8s.sh
+  - name: KUBE_TEST_REPO_LIST
+    value: /etc/repo-list/repo-list
+  volumes:
+  - name: azure-cred
+    secret:
+      secretName: azure-cred
+  - name: azure-ssh
+    secret:
+      secretName: azure-ssh
+  - name: repo-list
+    configMap:
```

Comment:
why do we need a configmap? can't we just put this file in one of the repos? it will probablybe easier to maintain this way

Reply:
yeah, makes sense to pull this from source so that it stays up to date?
IIRC: https://raw.githubusercontent.com/e2e-win/e2e-win-prow-deployment/master/repo-list.txt

Reply:
I'm all for that. As long as the proper env variable points to it, I'm happy with that. Basically, what this repo-list is: https://github.com/e2e-win/e2e-win-prow-deployment/blob/master/repo-list.txt . It needs to be used because we can't use upstream test images for Windows :) Works with this: https://github.com/kubernetes/kubernetes/pull/60848/commits/bb59d44deb11a87ca2c0dc88f955e06673192aa0

Reply:
sure, we can just specify a path in the env then? the checkout location should be stable so we can checkout e2e-win-prow-deployment and set this env to point at it?

Reply:
So yeah, dropped the configmap. The checkout, should I change kubetest to handle it ? Or just add deployment and provider check ( for acs-engine and azure ) here? : https://github.com/kubernetes/test-infra/blob/d4bb1da3cb8beabf061063221c2d90fb2b83764c/scenarios/kubernetes_e2e.py#L591

Or a third option I'm not thinking about ?

Reply:
the jobs could check it out, or kubetest could curl the file from raw.github?

or even just bake it into kubetest, who often is this file expected to change?

Reply:
how often*?

if it's not likely to change, I'd suggest just having kubetest write the contents to a tempfile and setting the environment as a super cheap way to plumb this through.

Reply:
It shouldn't change that much. The location at least shouldn't. Contents depends on the tests themselves ( i.e adding new repo ) or we eventually migrate from dockerhub. 

How about we do the same thing as we do for the windows-build-script [1]? Set the github location in the buildscripts, and then have kubetest download it and point the right variable to the path? 

This will protect against repo change ( say we move everything to sig-windows repo or smth like that somewhere along the lines ) because it's much easier/desirable to change variable in job than change kubetest. Kubetest just handles download of that file and points the env variable to it. 

[1] https://github.com/kubernetes/test-infra/pull/8037/commits/972a6f8bbe6caf08d364a8822f402ed8c980c29d#diff-62029f2ac8b693586f251fb0e1b73005R15

Reply:
forgot to reply back here explicitly (sorry!), kubetest was bumped with the changes from https://github.com/kubernetes/test-infra/pull/10190 so we're good to go!